.github/workflows/test_helper.yml

```diff
@@ -17,9 +17,30 @@ jobs:
     runs-on: ubuntu-22.04
     container:
       image: nycplanning/dev:${{ inputs.image_tag }}
+    env: 
+      BUILD_NAME: test
     steps:
       - uses: actions/checkout@v4
 
+      - name: Load Secrets
+        uses: 1password/load-secrets-action@v1
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          BUILD_ENGINE_SERVER: "op://Data Engineering/EDM_DATA/server_url"
+          BUILD_ENGINE_HOST: "op://Data Engineering/EDM_DATA/server"
+          BUILD_ENGINE_USER: "op://Data Engineering/EDM_DATA/username"
+          BUILD_ENGINE_PASSWORD: "op://Data Engineering/EDM_DATA/password"
+          BUILD_ENGINE_PORT: "op://Data Engineering/EDM_DATA/port"
+
+      - name: Set up dbt for sqlfluff templating
+        working-directory: ./
+        run: |
+          ./bash/build_env_setup.sh
+          cd products/ceqr_survey
+          dbt deps
```

Comment:
this works because `ceqr_survey` is the only getting `--templater=dbt`. I imagine this would have to be moved down to be with `export BUILD_ENGINE_DB=` when it has to be done for each proejct

Reply:
Yup. Or we'll need to have dbt dependencies specified at the repo level? Haven't yet looked into docs around multiple dbt projects in one repo